### PR TITLE
new menu structure + quick cleanup of the homepage

### PR DESCRIPTION
**Pull request by theute**
So here is a Pull request to see the new structure change "live"
I didn't go in details but the menu bar is pretty much the proposal made weeks ago.

### CODE REVIEW

**Comment by abonas** [comment]
a screenshot would help :)

File: src/main/jbake/content/clients/clients/index.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+= Clients
+Thoams Heute
```
Comment (abonas) [inline src/main/jbake/content/clients/clients/index.adoc]: typo ;)

**Comment by theute** [comment]
In theory there is a bot to build the website and make it available for 4 days, I hope it will work, it was down yesterday.

Thanks for noticing the typo

**Comment by abonas** [comment]
> In theory there is a bot to build the website and make it available for 4 days, I hope it will work, it was down yesterday.

thanks, didn't know that

**Comment by pilhuhn** [comment]
LGTM

**Comment by theute** [comment]
clients

Any better suggestion for this part ? It's repetitive and not clear

BTW, I plan to add ManageIQ as "client"

**Comment by pilhuhn** [comment]
On 8 Jul 2016, at 9:19, Thomas Heute wrote:

> clients
> 
> Any better suggestion for this part ? It's repetitive and not clear
> 
> BTW, I plan to add ManageIQ as "client"

What about only one (sub) menu point and then having a page that 
dispatches to both?

**Comment by theute** [comment]
That's a possibility indeed

**Comment by theute** [comment]
So the download page uses GitHub API and is not working on the auto-deployed instance because of rate limit on Red Hat IP... I don't know if Github hosted pages use the same limitation.

**Comment by theute** [comment]
Also limited there, so that last commit won't work and I need to revert it

**Comment by theute** [comment]
I screwed up with Git and not sure how to "fix" that... The result here is what I'd like to commit, but I see blog commits that should not be there I guess...

**Comment by theute** [comment]
Will try to recreate a proper PR... Wish me luck

**Pull request closed without merging**